batch/access-log-analyzer/src/main/java/AccessLogAnalyzer.java

```diff
@@ -69,19 +74,24 @@ public static void main(String[] args) throws Exception {
         DAG dag = new DAG();
         Vertex readFiles = dag.newVertex("readFiles", readFiles(sourceDir));
         Vertex parseLine = dag.newVertex("parseLine", map(LogLine::parse));
+        Vertex filterUnsuccessful = dag.newVertex("filterUnsuccessful",
```

Comment:
This sounds like it will pass only the unsuccessful responses. It should be `filterOutUnsuccessful`, or `filterOutFailed`, or `passSuccessful`.